qacode/core/utils/Utils.py

```diff
@@ -1,16 +1,23 @@
+# -*- coding: utf-8 -*-
+
+
 from sys import version_info
 from os import path
+from json import *
```

Comment:
No.

[Never use asterisk imports](https://pythonconquerstheuniverse.wordpress.com/2011/03/28/why-import-star-is-a-bad-idea/), no matter why (ok there are a few exceptions but if you're not sure if you should use it, you really shouldn't).

With that `from json import *` now you have declared in your global namespace the identifiers `dump`, `dumps`, `load`, `loads`, `JSONEncoder`, `JSONDecoder`, `JSONDecodeError`.

It's **much** better `import json` and then just use the methods you want (e.g. `json.loads`).